# message to update the artifact visualization update panel

uint8 data

uint8 DELETE = 0
uint8 UPDATE = 1
uint8 HIDE = 2
